app/views/devise/registrations/new.html.erb

```diff
@@ -6,6 +6,26 @@
   <% selected_primary_lang = nil %>
   <% secondary_lang_id = nil %>
 
+  <div class="field">
+    <%= f.label :fullname %><br />
+    <%= f.text_field :fullname, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :age %><br />
+    <%= f.text_field :age, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :gender %><br />
+    <%= f.text_field :gender, autofocus: true %>
```

Comment:
Change text_field to     

<%= form.select :gender, options_for_select([["Female", "female"], ["Male", "male"], ["Agender", "agender"], ["Genderfluid", "genderfluid"], ["Genderqueer", "genderqueer"], ["Transgender", "transgender"], ["Prefer Not to Say", "prefer not to say"]]) %> <br><br>

Reply:
You can leave the autofocus: true in there though